modules/ROOT/pages/asset-usage-and-engagement-metrics-dashboard.adoc

```diff
@@ -0,0 +1,78 @@
+= Asset Usage and Engagement Metrics Dashboard
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Exchange administrators can view usage and engagement metrics for API specification fragments and REST APIs using the dashboard. The dashboard provides a holistic view of asset usage and consumption within the organization. The usage metrics measure how many times an asset is downloaded from Exchange, imported from API Designer, imported from Anypoint Studio, and the number of dependents and contracts. The asset engagement score is a composite metric that administrators and asset owners can use to measure the usage of the assets in Exchange. The results for this engagement are the different consumption actions that end-users make across different points of need within Exchange. The engagement score reflects the usage of the asset in the context of the usage for API specification fragments and REST APIs in the customer root organization and business groups. 
+
+The dashboard displays the metrics for the top 50 assets for a period of the last 7 days, 30 days, or 60 days that are ranked by the most consumption events. You can filter the top assets by API specification fragments and REST API types and by one or more organizations. Export metric data into a CSV file by clicking CSV to Export.
```

Comment:
```suggestion
The dashboard displays the metrics for the top 50 assets, ranked by the most consumption events, for a period of the last 7 days, 30 days, or 60 days. You can filter the top assets by API specification fragments and REST API types and by one or more organizations. Export metric data into a CSV file by clicking CSV to Export.
```

Reply:
For "Export metric data into a CSV file by clicking CSV to Export." is there a corresponding procedure for this in the doc? If so, I would delete this because it's a step and should be in the format of steps.